### PR TITLE
Initialize LuaRocks library if not initialized

### PR DESCRIPTION
**Pull request by hishamhm**

### CODE REVIEW

File: datafile/openers/luarocks.lua [hunk 1]
```diff
@@ -16,6 +16,26 @@ end
 local load_local_manifest = manif_core.load_local_manifest  -- LuaRocks 2
 load_local_manifest = load_local_manifest or manif_core.fast_load_local_manifest  --LuaRocks 3
 
+local cfg
+ok, cfg = pcall(require, "luarocks.cfg") -- LuaRocks 2
+if not ok then
+   _, cfg = pcall(require, "luarocks.core.cfg") -- LuaRocks 3
+end
```
Comment (Tieske) [inline datafile/openers/luarocks.lua:23]: Shouldn't the checks be flipped?
```suggestion
ok, cfg = pcall(require, "luarocks.core.cfg") -- LuaRocks 3
if not ok then
   _, cfg = pcall(require, "luarocks.cfg") -- LuaRocks 2
end
```
LuaRocks 3 if more likely to have a stray LuaRocks 2 file around, than the other way around. Just to be safe.

**Review by Tieske — APPROVED** [review]
minor comment, other than that, lgtm

**Comment by hishamhm** [comment]
That's good feedback! But there is already another check in place above in
that module which checks for LR2 first, so I followed the example to avoid
any changes in behavior ("if nobody complained, assume it ain't broke, so
don't fix it" :) ) -- I was going for a minimal-disruption approach in this
one.

File: datafile/openers/luarocks.lua (hunk 1 above)
Comment (Tieske) [reply datafile/openers/luarocks.lua:23]: I missed the check above. Nvm.

**Pull request merged**